node/pegnet/addresses.go

```diff
@@ -462,6 +462,23 @@ func (p *Pegnet) IsIncludedTopPEGAddress(address []byte) bool {
 	return false
 }
 
+func (p *Pegnet) IsNonZeroPEGAddress(address []byte) bool {
+	stmt2 := `SELECT COUNT(*) FROM pn_addresses WHERE peg_balance > 0 AND address = ?;`
+	rows, err2 := p.DB.Query(stmt2, address[:])
```

Comment:
address[:] is unnecessary

Reply:
Ok, updated the `IsNonZeroPEGAddress ()` function.